pkg/config/config.go

```diff
@@ -343,15 +369,6 @@ func newSystemChannelGroup(channelConfig Channel) (*cb.ConfigGroup, error) {
 		return nil, err
 	}
 
-	if channelConfig.Consortium == "" {
-		return nil, errors.New("consortium is not defined in channel config")
-	}
```

Comment:
This check probably should move to `newChannelGroup`

Reply:
Yes, I have fixed it and added an unit test case.